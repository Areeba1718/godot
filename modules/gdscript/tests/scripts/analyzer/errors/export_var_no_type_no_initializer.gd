# GH-74188

@export_multiline var text

func test():
	pass
