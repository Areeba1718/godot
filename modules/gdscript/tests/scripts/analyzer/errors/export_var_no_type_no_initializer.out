GDTEST_ANALYZER_ERROR
Cannot use annotation "@export_multiline" with variable without a specified type or initializer.
